spec/models/federated_protocols/saml_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+module FederatedProtocols
+  RSpec.describe Saml do
+    let(:authn_contexts) { [] }
+    let(:saml_request) { instance_double(SamlIdp::Request) }
+    subject { FederatedProtocols::Saml.new saml_request }
+    before do
```

Comment:
[nitpick] can you please add a newline between `let`/`subject` statements and `it`/`before` blocks? it makes tests a lot more readable. thanks!

Reply:
I'll try and remember this next time :-)